external-api/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts

```diff
@@ -67,18 +67,19 @@ export const eStoreTermStoreAndBuyerFolder = async (eStoreData: any) => {
     if (buyerFolderResponse?.status === 201) {
       console.info('API Call finished: The Buyer folder for %s was successfully created', eStoreData.buyerName);
 
+      const tfmDealsCollection = await getCollection('tfm-deals');
+      // update the `tfm-deals` collection once the buyer and deal folders have been created
```

Comment:
Should the comment say
```suggestion
      // update the `tfm-deals` collection once the buyer folder has been created
```
instead?

Reply:
good spot. I've updated that now